### ISSUE TITLE
Feature request ability to load own music and sound fx files.

### ISSUE THREAD

**Issue by RetroGamer02** [skynettx/raptor#16]
I was thinking this would let you use the audio from say the 2010/2015 release of the game or even remixes!

**Comment by darkstar252** [comment]
I agree but this will need ./music folder (at least for linux) But for me is better to use a compressed format (eg. rar or lzh) to decompress in memory and load the music(like eduke32 it has).

With this thing is easy to record the original OPL3 music in an ogg file compress it to rar lzh or whatever you want and load if the configuration is for sound blaster , and record music using ROLAND SC-55 and the same thing in ogg format and compressed onto an rar or lzh , and load this file in memory to load the music if there is a General Midi config in the setup.
